### PR TITLE
ternary contour notebook

### PR DESCRIPTION
**Pull request by emmanuelle**
New notebook for recent figure factory.

### CODE REVIEW

**Comment by emmanuelle** [comment]
Should we also keep the example of https://plot.ly/python/ternary-contour/?

**Comment by nicolaskruchten** [comment]
Sorry this dropped off my radar! I think this notebook should replace the existing one, which doesn't actually do contours. 

You will probably need to rebase this PR onto `master` and update `requirements.txt` to include `scikit-image` so that CI will pass :)

thanks!

**Comment by nicolaskruchten** [comment]
@emmanuelle can we try to revive this PR too please?

**Comment by nicolaskruchten** [comment]
The build is failing, and the output is in the "artifacts" under "failures": https://209-189674195-gh.circle-artifacts.com/0/build/failures/ternary-contour

**Comment by nicolaskruchten** [comment]
If you could rebase this onto `master`, it would clean up the commit history a bit :) We're big rebasing fans in general at Plotly (except in the Plotly.js repo)

**Comment by emmanuelle** [comment]
yes it looks better :-)

File: python/ternary-contour.md [hunk 1]
```diff
@@ -22,63 +22,127 @@ jupyter:
     pygments_lexer: ipython3
     version: 3.6.7
   plotly:
-    description: How to make Ternary Contour Plots in Python with Plotly.
-    display_as: scientific
+    description: How to make ternary contour plots with plotly
```
Comment (nicolaskruchten) [inline python/ternary-contour.md]: I think the previous title was good actually

File: python/ternary-contour.md [hunk 2]
```diff
@@ -22,63 +22,127 @@ jupyter:
...
+    display_as: 3d_charts
```
Comment (nicolaskruchten) [inline python/ternary-contour.md]: this isn't really 3d in the 3d sense :)

File: python/ternary-contour.md [hunk 3]
```diff
@@ -22,63 +22,127 @@ jupyter:
...
     has_thumbnail: true
-    ipynb: ~notebook_demo/40
     language: python
     layout: user-guide
-    name: Ternary Contour Plots
-    order: 10
+    name: Ternary contours
     page_type: u-guide
-    permalink: python/ternary-contour/
+    permalink: python/ternary-contours/
```
Comment (nicolaskruchten) [inline python/ternary-contour.md]: changing the URL actually breaks things so let's leave this alone for now. We already have some search-engine ranking on this page, and the matching R and JS pages have the same URL so the auto-linking already works etc.

**Comment by emmanuelle** [comment]
The discrepancies you found came from the fact that I had started a new file from scratch, I had forgotten that a stub already existed

**Comment by nicolaskruchten** [comment]
💃 !

**Pull request merged**